### PR TITLE
LPS-93281 - Asset framework dev

### PR DESCRIPTION
**Pull request by stephenkostas**

### CODE REVIEW

**Comment by stephenkostas** [comment]
https://issues.liferay.com/browse/LPS-93281

File: developer/frameworks/articles/05-assets/05-relating-assets.markdown [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+        long insultId = ParamUtil.getLong(renderRequest, "insultId");
+        Insult ins = InsultLocalServiceUtil.getInsult(insultId);
+        AssetEntry assetEntry = AssetEntryLocalServiceUtil.getEntry(Insult.class.getName(), ins.getInsultId());
+        %>
+
+2.  Use the `liferay-asset:asset-links` tag to show the entity's related assets.
+    For this tag, you should retrieve the entity's class name and the variable
+    holding your instance object, so you can return its ID. The example code
+    below uses the example entity class `Insult` and an instance object
+    variable called `ins`:
```
Comment (jkappler) [inline developer/frameworks/articles/05-assets/05-relating-assets.markdown]: @stephenkostas, I don't see the entity class used in the example.

**Comment by jkappler** [comment]
@stephenkostas, all looks well, only the comment I made. Let me know please. And thanks for this!

File: developer/frameworks/articles/05-assets/05-relating-assets.markdown (hunk 1 above)
Comment (stephenkostas) [reply developer/frameworks/articles/05-assets/05-relating-assets.markdown]: I think that description slipped through the cracks on a couple updates. I just updated the branch with this text: 
" For this tag, you retrieve the `assetEntryId` from the `assetEntry` object, 
    retrieve your asset's `className`, and get the entity's primary key 
    (`classPK`) from the specific `entry`. The tag then retrieves any other 
    assets linked to your asset."

Comment (jkappler) [reply developer/frameworks/articles/05-assets/05-relating-assets.markdown]: Looks good @stephenkostas, thanks!

Comment (stephenkostas) [reply developer/frameworks/articles/05-assets/05-relating-assets.markdown]: Thanks!

**Pull request closed without merging**